src/history/hs.c

```diff
@@ -917,6 +948,8 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi)
              * If the time points are out of order (which can happen if the application performs
              * updates with out-of-order timestamps), so this value can never be seen, don't bother
              * inserting it.
+             *
+             * FIXME-WT-6388: We should be able to replace this with an assertion.
```

Comment:
If we tried to do that right now will the assertion fire?
I prefer to create a new ticket to track a change, and refer to that in the FIXME.

Reply:
I think it can only fire in the case of out-of-order transaction ids.
I'm happy to make a ticket to do this separately.

Reply:
0143cd0